libraries/matchers.rb

```diff
@@ -1,8 +1,8 @@
 #
 # Author:: Shawn Neal (<sneal@sneal.net>)
-# Cookbook Name:: visualstudio
+# Cookbook:: visualstudio
```

Comment:
This looks like a copy pasta comment error with regards to the cookbook name (I know it's not to do with this PR specifcally just might want to fix it as it's a bit confusing.